style.css

```diff
@@ -48,11 +48,62 @@ body {
   }
 }
 
-.material-icon {
+.menu-icon {
   width: 24px;
   height: auto;
 }
 
+.hamburger {
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  padding: 0;
+  border: #fff;
+
+  /* background: white; */
+  cursor: pointer;
+  color: #6070ff;
+}
+
+.close-icon {
+  display: none;
+  background: #6070ff;
+  color: #fff;
+}
+
+.menu {
+  position: absolute;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #6070ff;
+  opacity: 0.9;
+  color: white;
+  list-style: none;
+  padding-top: 4rem;
+  padding-left: 2rem;
+  gap: 12px;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
```

Comment:
[OPTIONAL] You did a great job with your mobile menu, well done 😄! I liked the animation effect as well. Please take into account that most of the mobile menus stick to the user's screen. This means that, when scrolling down, the menu is still being displayed at the top of the page. Currently, your menu is static on the beginning of your page, causing this effect when scrolling down:
![image](https://user-images.githubusercontent.com/42799579/201729383-e9620b9e-6866-460d-a242-cd6a765e2fd0.png)

So, to improve the UX/UI of your webpage, kindly consider this advice 😄.

> You could do it by giving a position: fixed to your menu 😉.

Reply:
Very educational👌. Thank you so much